Source/Scene/Cesium3DTileset.js

```diff
@@ -2050,6 +2049,44 @@ function processMetadataExtension(tileset, tilesetJson) {
   });
 }
 
+/**
+ * If tileset metadata is present, initialize the {@link Cesium3DTilesetMetadata}
+ * instance. This is asynchronous since metadata schemas may be external.
+ *
+ * @param {Cesium3DTileset} tileset The tileset
+ * @param {Object} tilesetJson The tileset JSON
+ * @return {Promise<Object>} A promise that resolves to tilesetJson for chaining.
+ * @private
+ */
+function processMetadataExtension(tileset, tilesetJson) {
```

Comment:
After looking at the change I suggest in `Cesium3DTilesetMetadata`, see if anything else can be simplified here between the two functions. It looks like the steps are close to identical, it's mainly the input that differs